lang/cs/Org.Apache.REEF.Network/Elastic/Operators/Logical/Default/DefaultOneToN.cs

```diff
@@ -0,0 +1,238 @@
+            offset += sizeof(ushort);
+
+            switch (msgReceived)
+            {
+                case TaskMessageType.JoinTopology:
+                    {
+                        var operatorId = BitConverter.ToInt16(message.Message, offset);
+
+                        if (operatorId != _id)
+                        {
+                            return false;
+                        }
+
+                        if (!Stage.IsCompleted && _failureMachine.State.FailureState < (int)DefaultFailureStates.Fail)
+                        {
+                            var taskId = message.TaskId;
+                            Log.Log(Level.Info, "{0} joins the topology for operator {1}", taskId, _id);
+
+                            _topology.AddTask(taskId, _failureMachine);
+                        }
+
+                        return true;
+                    }
+                case TaskMessageType.TopologyUpdateRequest:
+                    {
+                        var operatorId = BitConverter.ToInt16(message.Message, offset);
+
+                        if (operatorId != _id)
+                        {
+                            return false;
+                        }
+
+                        Log.Log(Level.Info, "Received topology update request for {0} {1} from {2}",
+                            OperatorType.ToString(), _id, message.TaskId);
+
+                        _topology.TopologyUpdateResponse(
+                            message.TaskId,
+                            ref returnMessages,
+                            Optional<IFailureStateMachine>.Of(_failureMachine));
+
+                        if (_stop)
+                        {
+                            if (_failureMachine.State.FailureState < (int)DefaultFailureStates.StopAndReschedule)
+                            {
+                                _stop = false;
+                            }
+                            else
+                            {
+                                returnMessages.Clear();
+                                Log.Log(Level.Info, "Operator {0} is in stopped: Waiting.",
+                                    OperatorType.ToString());
+                            }
+                        }
+
+                        return true;
+                    }
+                case TaskMessageType.CompleteStage:
+                    {
+                        _failureMachine.Complete();
+                        Stage.Complete();
+
+                        return true;
+                    }
+
+                default:
+                    return false;
+            }
+        }
+
+        /// <summary>
+        /// Mechanism to execute when a reconfigure event is triggered.
+        /// <paramref name="reconfigureEvent"/>
+        /// </summary>
+        public override void OnReconfigure(ref ReconfigureEvent reconfigureEvent)
+        {
+            Log.Log(Level.Info, "Going to reconfigure the {0} operator", OperatorType.ToString());
```

Comment:
no need to call `.ToString()` explicitly

Reply:
Fixed.